va-gov/layouts/home.html

```diff
@@ -34,7 +34,7 @@ <h3 class="hub-links-title"><i class="icon-large fa fa-handshake-o hub-color-dis
             <li><a href="/claim-or-appeal-status">Check your claim or appeal status</a></li>
             <li><a href="/va-payment-history/">View your payment history</a></li>
             <li><a href="/disability/upload-supporting-evidence/">Upload evidence to support your claim</a></li>
-            <li><a href="/disability-benefits/apply/form-526-disability-claim">File for a VA disability increase</a></li>
+            <li><a href="/disability/how-to-file-claim/">File for a VA disability increase</a></li>
             <li><a href="/disability/how-to-file-claim/">File a claim for compensation</a></li>
```

Comment:
Should these two links both be `/disability/how-to-file-claim/`?

Reply:
I asked the same question, and @djmunoz1 said yes. :man_shrugging: